### PR TITLE
Support for [, drop = TRUE] for grouped data frames

### PR DESCRIPTION
**Pull request by romainfrancois**
Most of the work is already done by tibble:::[.tbl_df

### CODE REVIEW

File: R/grouped-df.r [hunk 1]
```diff
@@ -95,12 +95,16 @@ ungroup.grouped_df <- function(x, ...) {
   ungroup_grouped_df(x)
 }
 
+#' @importFrom tibble is_tibble
 #' @export
-`[.grouped_df` <- function(x, i, j, ...) {
+`[.grouped_df` <- function(x, i, j, drop = FALSE) {
```
Comment (krlmlr) [inline R/grouped-df.r]: `?"[.data.frame"` has no ellipsis, but in `?"["` the ellipsis is part of the signature. I'd prefer keeping it before the `drop` argument, if only to ensure that `drop` is named.

Comment (romainfrancois) [reply R/grouped-df.r]: In that case, please lead the way and make the change in `tibble` first: https://github.com/tidyverse/tibble/blob/1810b000a8e9ca39b3f3f68927efde44f0d47bd3/R/tbl-df.r#L58

Comment (hadley) [reply R/grouped-df.r]: I think we should match [.data.frame since [ is primitive and only does arg matching by position.

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**